README.md

```diff
@@ -186,9 +187,37 @@ fmt.Println(formattedStr)
 //   main.readFile | .../example/main.go | 6
 ```
 
-### Writing your own custom format
+### Writing custom output format
```

Comment:
minor: i would change this to either `Writing custom output formats` or `Writing a custom output format`